components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -149,7 +149,9 @@ BraveReferralsService::BraveReferralsService(PrefService* pref_service,
           base::ThreadPool::CreateSequencedTaskRunner({base::MayBlock()})),
       pref_service_(pref_service),
       api_key_(api_key),
-      platform_(platform) {}
+      platform_(platform) {
+  pref_service_->ClearPref(kSafetynetStatus);
```

Comment:
@simonhong clearing the pref here as we were mainly using the pref in the class.